tests/pyrog/api/test_source.py

```diff
@@ -26,6 +28,33 @@ def test_retrieve_source(api_client, source):
     assert response.status_code == 200
 
 
+def test_create_full_source(api_client, exported_source):
+    url = reverse("sources-list")
+
+    response = api_client.post(url + "?full=True", exported_source, format="json")
+
+    assert response.status_code == 201
+    assert Source.objects.count() == 1
+    assert Resource.objects.count() == 5
+    assert Attribute.objects.count() == 35
+    assert InputGroup.objects.count() == 15
+    assert Input.objects.count() == 16
+    assert Column.objects.count() == 27
+    assert Condition.objects.count() == 3
+    assert Filter.objects.count() == 1
+
```

Comment:
missing assertion on Joins

Reply:
also, I feel that counting the inserted objects is quite limited (are we sure the joins were created in the right order for instance ?).
Do you think we could do something like: 
- create a full source
- retrieve it
- compare the serialization of the retrieved source with the initial fixture

you'd have to remove the IDs from the Column and Owner objects for the serializations to be equal but I doubt the arrays would have a consistent order across import/exports. What do you think ?